Source/debug.cpp

```diff
@@ -565,7 +565,7 @@ std::string DebugCmdLevelUp(const std::string_view parameter)
 	Player &myPlayer = *MyPlayer;
 	std::string cmdLabel = "[givelvl] ";
 
-	const int levels = ParseInt<int>(parameter, /*min=*/1).value_or(1);
+	const int levels = std::min(ParseInt<int>(parameter, /*min=*/1).value_or(1), 50);

```

Comment:
```suggestion
	const int levels = std::min<int>(ParseInt<int>(parameter, /*min=*/1).value_or(1), GetMaximumCharacterLevel());
```

Reply:
Done, thanks, didn't see that macro, was looking for defines or constants :)